tests/app/notifications/rest/test_send_notification.py

```diff
@@ -142,18 +142,18 @@ def test_send_notification_with_placeholders_replaced(notify_api, sample_email_t
         ['Jo', 'John', 'Josephine'],
         (
             'Hello \n\n'
-            '* Jo\n'
-            '* John\n'
-            '* Josephine\n'
-            'This is an email from GOV.UK'
+            '• Jo\n'
+            '• John\n'
+            '• Josephine\n'
+            'This is an email from GOV.\u200bUK\n'
```

Comment:
Could you explain why you are making all the unicode changes in these tests please as I'm not sure why?

Reply:
I’ve found a way of doing this without the unicode changes now (they are replacements we make specifically for emails to stop browsers making GOV.UK into a link).